api/environments/dynamodb/dynamodb_wrapper.py

```diff
@@ -85,22 +85,25 @@ def search_items_with_identifier(
             query_kwargs.update(ExclusiveStartKey=start_key)
         return self.query_items(**query_kwargs)
 
-    def get_migration_status(self, project_id: int) -> str:
+    def get_migration_status(self, project_id: int) -> ProjectIdentityMigrationStatus:
```

Comment:
I don't quite understand why this method is on the DynamoIdentitiesWrapper (which seems to hold logic for working with the Identities table)? Since this method is not using any attributes from the class, we should either move it elsewhere or declare it as a staticmethod. My preference would be the former. I kind of think that this should just be a property on the `DynamoProjectMetadata` class itself.

Reply:
The Idea behind putting this method on `DynamoIdentitiesWrapper` was to hide the existence of `DynamoProjectMetadata` from everything except `DynamoIdentitiesWrapper`, and for it to exists only as an implementation detail of `DynamoIdentitiesWrapper`.